consoleview.go

```diff
@@ -91,29 +152,31 @@ func fatalf(format string, args ...interface{}) {
 	os.Exit(1)
 }
 
-func printWrappedText(text string, width, subsequentIndent int) {
+func printWrappedText(text string, width, subsequentIndent int) string {
+	var wrappedText bytes.Buffer
 	tokens := strings.Split(text, " ")
 	offset := 0
 	for i, token := range tokens {
 		if i > 0 && offset+len(token) > width {
-			fmt.Printf("\n%s", strings.Repeat(" ", subsequentIndent))
+			wrappedText.WriteString(fmt.Sprintf("\n%s", strings.Repeat(" ", subsequentIndent)))
```

Comment:
That's a common Go error. It just means you need a pointer rather than a value: `fmt.Fprintf(&wrappedText, ...)`